verticapy/tests_new/performance/vertica/collection/test_export_simple.py

```diff
@@ -0,0 +1,137 @@
+"""
```

Comment:
@vikash018 please look at this file and tell us if it works for you

Reply:
@jslaunwhite-microfocus - As we discussed in call, we need to add following UT
1) Row count validation for import/export rows
2) Data content validation
3) Interval data validation for imported rows

Pls feel free to update above list for missing tests.

Reply:
I added a simple row count validation for import/export. A previous PR already added some tests for import, but things changed enough that it is worth checking again.